### ISSUE TITLE
sometimes text runs off the screen

### ISSUE THREAD

**Issue by fdncred** [amtoine/nu_plugin_explore#27]
Sometimes the text runs off the screen to the right. It would be nice to detect this and add scrolling left and right.

This is a screenshot of `sys | nu_plugin_explore` and then hitting `l` on the `cpu` row.

![image](https://github.com/amtoine/nu_plugin_explore/assets/343840/15ca7125-8a76-4c23-93bc-b3dccf14ee52)

**Comment by amtoine** [comment]
yeah this is an issue i have had since the beginning, thanks for reporting this :pray: 

this has to do with the table rendering and might be related to the 2D navigation in tables i've already filed in #19 :yum:
